pkg/utils/openshift/utils.go

```diff
@@ -51,3 +57,54 @@ func MapKnownVersion(info platform.PlatformInfo) platform.OpenShiftVersion {
 	}
 	return platform.OpenShiftVersion{Version: k8sToOcpMap[info.K8SVersion]}
 }
+
+func GetConsoleYAMLSample(res resource.KubernetesResource) (*consolev1.ConsoleYAMLSample, error) {
```

Comment:
Please create your own webconsole.go file in this same folder/package. Same goes for unit tests.

Reply:
modified